### ISSUE TITLE
Screen capture area selection & ability to reset area without restarting

### ISSUE THREAD

**Issue by pinntokuru** [AuroraWright/owocr#1]
Currently you need to input coordinates and height/width for owocr to scan a particular area of the screen. It would be much more convenient if it was possible to simply drag a rectangle instead.

Another quality of life change would be the ability to change these coordinates (also via mouse drag) with a hotkey without having to close and restart the app.

**Comment by AuroraWright** [comment]
I agree but it’s very annoying to implement unfortunately, i tried to mess around with a pyqt example last year at first but i couldn’t get it to work with two monitors and it had some other issue so i gave up

**Comment by pinntokuru** [comment]
Instead of a full on drag and drop having the user click twice and calculating an area using those two points might be enough.

**Comment by ratlair** [comment]
+1
